docs/spark/tutorials/databricks-deployment.md

```diff
@@ -9,161 +9,193 @@ ms.custom: mvc
 
-[Databricks](https://databricks.com) is a platform that provides cloud-based big data processing using Apache Spark.
+2. The install-worker.sh is a script that lets you copy .NET for Apache Spark dependent files into the nodes of your cluster. 
 
-> [!NOTE]
-> [Azure Databricks](https://azure.microsoft.com/services/databricks/) and [AWS Databricks](https://databricks.com/aws) are Linux-based. Therefore, if you are interested in deploying your app to Databricks, make sure your app is .NET Standard compatible and that you use [.NET Core compiler](https://dotnet.microsoft.com/download) to compile your app.
+   Create a new file named **install-worker.sh** your local computer, and paste the [install-worker.sh contents located on GitHub](https://raw.githubusercontent.com/dotnet/spark/master/deployment/install-worker.sh). 
 
-Databricks allows you to submit .NET for Apache Spark apps to an existing active cluster or create a new cluster every time you launch a job. This requires the **Microsoft.Spark.Worker** to be installed before you submit a .NET for Apache Spark app.
+3. The db-init.sh is a script that installs dependencies onto your Databricks Spark cluster.
 
-### Deploy Microsoft.Spark.Worker
+   Create a new file named **db-init.sh** your local computer, and paste the [install-worker.sh contents located on GitHub](https://github.com/dotnet/spark/blob/master/deployment/db-init.sh). 
+   
+   In the file you just created, set the DOTNET_SPARK_RELEASE variable to `https://github.com/dotnet/spark/releases/download/v0.3.0/Microsoft.Spark.Worker.netcoreapp2.1.linux-x64-0.3.0.tar.gz`. Leave the rest of the db-init.sh file as-is.
 
-This step is only required once for a cluster.
+> [!Note]
+> If you are using Windows, verify that the line-endings in your **install-worker.sh** and **db-init.sh** scripts are Unix-style (LF). You can change line endings through text editors like Notepad++ and Atom.
 
-1. Download [db-init.sh](https://github.com/dotnet/spark/blob/master/deployment/db-init.sh) and [install-worker.sh](https://github.com/dotnet/spark/blob/master/deployment/install-worker.sh
-) onto your local machine.
+## Publish your app
 
-2. Modify **db-init.sh** to point to the **Microsoft.Spark.Worker** release you want to download and install on your cluster.
+Next, you publish the *mySparkApp* created in the [.NET for Apache Spark - Get Started in 10-Minutes]() tutorial to ensure your Spark cluster has access to all the files it needs to run your app. 
 
-3. Install the [Databricks CLI](https://docs.databricks.com/user-guide/dev-tools/databricks-cli.html).
+1. Run the following commands to publish the *mySparkApp*.
 
-4. [Setup authentication](https://docs.databricks.com/user-guide/dev-tools/databricks-cli.html#set-up-authentication) details for the Databricks CLI.
+   **On Windows:**
 
-5. Upload the files to your Databricks cluster using the following command:
+   ```cmd
```

Comment:
@mairaw, should this be `console` or `dotnetcli` ?

Reply:
Same comment as in the other PR